spec/services/slack_user_finder_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+describe SlackUserFinder do
+  describe "#user_exists?" do
+    it "returns true if a user is found" do
+      slack_username_from_fixture = "testusername"
+
+      user_exists = SlackUserFinder.new(
+        slack_username_from_fixture,
+        Slack::Web::Client.new
+      ).user_exists?
+
+      expect(user_exists).to be true
+    end
+
+    it "returns false if a user was not found" do
+      fake_slack_user_name = "testusernamethatdoesnotexist"
+
+      user_exists = SlackUserFinder.new(
+        fake_slack_user_name,
+        Slack::Web::Client.new
+      ).user_exists?
+
+      expect(user_exists).to be false
```

Comment:
Oh interesting, thank you for the heads up with the RSpec info (something I'm still diving into).  As for the name, are you thinking that it should be changed in the `EmployeeFinder` service class?  I'm not sure about `is_in_organization?` only because that sounds like it should accept an argument of an organization to me and we're just checking for the existence of a user in the organization we're a part of when logged in.  I agree that the naming could be improved though, especially in light of how it flows with the Rspec matchers.